canvas_site_deletion/views.py

```diff
@@ -91,9 +94,19 @@ def delete(request, pk):
     canvas_course_id = None
     try:
         ci = CourseInstance.active_and_deleted.get(course_instance_id=pk)
+
         canvas_course_id = ci.canvas_course_id
         ts = int(time.time())
 
+        current_tool_launch_account_id = int(request.LTI["custom_canvas_account_id"])
+
+        account_ids = get_account_hierarchy(ci.canvas_course_id)
+
+        if current_tool_launch_account_id not in account_ids:
+            logger.error(f'User {request.user} is trying to delete the Canvas site associated with course_instance {pk} but is not in the correct account or sub-account where they have launched the site deletion tool.')
```

Comment:
Let's add the account_ids list to the log message too so that it's easier to debug. You can also add a second argument to the `logger.error()` call like this: 
```python
logger.error(f"error message...", extra={"course_instance_id": pk, "account_ids": account_ids});
```